mmcv/ops/csrc/pytorch/cuda/active_rotated_filter_cuda.cu

```diff
@@ -14,7 +14,7 @@ void ActiveRotatedFilterForwardCUDAKernelLauncher(const Tensor input,
   int kW = input.size(4);
   int num_rotations = indices.size(3);
   int nEntry = num_orientations * kH * kW;
-  int output_size = output.numel();
+  int output_size = input.numel();
```

Comment:
Why should the output_size equal `input.numel()` rather than `output.numel()`?

Reply:
We are just following the original implementation, which uses variable name output_size as well. https://github.com/csuhan/s2anet/blob/d2870963815b10c8f7796577e30b7c3b14e47ac6/mmdet/ops/orn/src/cuda/ActiveRotatingFilter_cuda.cu#L94
Actually the numel of `output` tensor is 8 times the numel of `input`. Because the `output` feature tensor is orientation-sensitive and it encodes the features for all 8 discretized orientations.

Reply:
Got it.